### PR TITLE
Add instructions to install via Chocolatey

### PR DESCRIPTION
**Pull request by chantisnake**

### CODE REVIEW

**Comment by matheuss** [comment]
@chantisnake do we need to update the latest version on chocolatey whenever we release a new one?

**Comment by chantisnake** [comment]
no, I will take care of it.

**Comment by matheuss** [comment]
Hm, so we do have 😅 Thank you ❤️

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (matheuss) [inline README.md]: Could you remove this line? 😅

**Comment by chantisnake** [comment]
It's the line I removed the right line?

**Comment by matheuss** [comment]
It is @chantisnake. Thank you! 🙌

**Pull request merged**